sso/frontend/src/App.tsx

```diff
@@ -1,12 +1,33 @@
-import { Login } from './components/organisms/Login'
-import React from 'react'
+import DashboardPage from './components/organisms/dashBoard';
+import { Login } from './components/organisms/Login';
+import React, { useEffect, useState } from 'react';
+import { BrowserRouter, Navigate, Route, Routes } from 'react-router-dom';
+import { isExpired } from 'react-jwt';
+import { useAuth0 } from '@auth0/auth0-react';
 
 const App = () => {
+  const { isAuthenticated } = useAuth0();
+  const [token, setToken] = useState<string>('');
+  useEffect(() => {
+    const token: string = localStorage.getItem('token')!;
+    setToken(token);
+  }, [token, isAuthenticated]);
+  function isExpired(token: any) {
+    throw new Error('Function not implemented.');
+  }
+
```

Comment:
don't include business logic in App.tsx, move this inside the page or in a custom hook